### PR TITLE
Add multi-Terraform workspace example

### PR DESCRIPTION
**Pull request by aliscott**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (hugorut) [inline .vscode/settings.json:1]: do we actually want to commit this? Or just keep locally?

**Review by hugorut — APPROVED** [review]
Not sure if you meant to commit the settings.json. I don't have a massive feeling against keeping it in or out tho. I'll leave the decision to you.

File: .vscode/settings.json (hunk 1 above)
Comment (aliscott) [reply .vscode/settings.json:1]: I don't have a strong opinion. I see three options:
1. Commit it
2. Don't commit it
3. Move it to `.vscode/settings.json.default` or some other file so others have a reference of it.

@vdmgolub might have input as well.

Comment (vdmgolub) [reply .vscode/settings.json:1]: If I understand the issue origin correctly, I would fix the trailing spaces issue, I don't think it's a good thing to have golden files with them. This setting will allow more trailing spaces in future, which will make the issue worse. So I would vote for 2 as eventually redundant.

File: examples/multi-terraform-workspace/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      vmImage: ubuntu-latest
+
+    steps:
+      - task: TerraformInstaller@0
+        displayName: Install Terraform
+
+      - task: InfracostSetup@0
+        displayName: Setup Infracost
+        inputs:
+          apiKey: $(apiKey)
```
[the remark this answers is not in the document]
Comment (aliscott) [reply examples/multi-terraform-workspace/README.md:23]: 👍  done

**Pull request merged**